source/agora/flash/Node.d

```diff
@@ -374,6 +405,12 @@ public abstract class FlashNode : ControlFlashAPI
 
         foreach (_, chan; this.channels)
             this.network.addChannel(chan.conf);
+
+        if (this.conf.listener_address.length != 0)
+            this.listener = this.getFlashListenerClient(
+                this.conf.listener_address, Duration.init);
+        else  // avoid null checks & segfaults
+            this.listener = new BlackHole!FlashListenerAPI();
```

Comment:
Notes about this:

- Either we use the code in the PR
- Or we make `listener` optional and we always do `&& this.listener !is null` everywhere.
- Or we make `listener` a required option so it must not be null.